src/main/java/Steven.java

```diff
@@ -0,0 +1,62 @@
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+
+import mainFiles.Parser;
+import mainFiles.Storage;
+import mainFiles.TaskList;
+import mainFiles.UserInput;
+
+/**
+ * Main code for running Steven
+ */
+public class Steven {
+
+    private static final String line = "========\n";
+    private static final String bootMsg = ("This is Steven!\nHow can I advise?\n");
```

Comment:
Do you think it would be more aligned with the style guide to name the variable messageWhenBooting? I feel that would be more noun like